10_cdisc_questions.md

```diff
@@ -0,0 +1,68 @@
+1. What is the main focus of Good Clinical Practices (GCP)? [comment]: <> (This is not appropriate for CDISC section, perhaps could be moved general programming section)
+    A. harmonized data collection
+    B. standard analysis practices
+    C. protection of subjects
+    D. standard monitoring practices
+    *** Answer: C. protection of subjects*** 
+
+ 2. Specify the variable role which focus on the observation (such as the name of a lab test) of a SDTM data set? 
+    A. Identifier
+    B. Topic
+    C. Timing
+    D. Qualifier
+    Answer: B. Topic
+
+ 3. Concomitant/Prior Medications (CM) is a component of which SDTM class?
+    A. Findings
+    B. Interventions
+    C. Events
+    D. Special Purpose
+    Answer: B. Interventions
+
+ 4. What is the primary purpose of programming validation? [comment]: <> (This is not appropriate for CDISC section, perhaps could be moved general programming section)
+    A. Ensure that the output from both the original program and the validation program match.
+    B. Efficiently ensure any logic errors are discovered early in the programming process.
+    C. Justify the means used to accomplish the outcome of a program and ensure its accurate representation of the original data.
+    D. Document all specifications pertaining to programmed output and ensure all were reviewed during the programming process.
+    Answer: C. Justify the means used to accomplish the outcome of a program and ensure its accurate representation of the original data.
+
+ 5. Why is the Demographics (DM) domain required for all human clinical trials regulatory submissions?

```

Comment:
28Sep: Perhaps remove detailed questions like this and replace with a general question, which one these is SDTM domain?

Reply:
Can we add some of these to the MD file? I accepted the merge